tests/integration/test_full_pipeline.py

```
@@ -0,0 +1,410 @@
+
+5. NO DISPUTE RESOLUTION
+   All disputes decided solely by Client.
+"""
+        
+        contract_file = tmp_path / "high_risk.txt"
+        contract_file.write_text(high_risk_contract)
+        
+        orchestrator = ReviewOrchestrator()
+        
+        with patch('builtins.input', return_value='y'):
+            result = orchestrator.review_contract(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)